client/imports/labpage/Lab

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html>
+<head>
+<style>
+#header {
+    background-color:black;
+    color:white;
+    text-align:center;
+    padding:5px;
+}
+#terminal{
+    width:350px;
+    float:left;
+    padding:70px;
+    border-left:5px solid #38546d; 
+
+
+}
+#nav {
+    line-height:30px;
+    background-color:#eeeeee;
+    height:300px;
+    width:100px;
+    float:left;
+    padding:5px;
+
+}
+#section {
+    width:350px;
+    float:left;
+    padding:10px;
+}
+#footer {
+    background-color:black;
+    color:white;
+    clear:both;
+    text-align:center;
+    padding:5px;
+}
+</style>
+</head>
+<body>
+
+<div id="header">
+<h1>15131:Great Practical Ideas in CS</h1>
+</div>
+
+<div id="nav">
+Task1<br>
+Task2<br>
+Task3
+</div>
+
+
+
+<div id="section">
```

Comment:
This should be where the Markdown component is injected- so that people can view the current task.  Plus, there should be next task and task status indicators visible from the data.